src/main/java/com/gregtechceu/gtceu/data/recipe/misc/WoodMachineRecipes.java

```diff
@@ -43,268 +46,282 @@ public static void init(Consumer<FinishedRecipe> provider) {
-                            .log(GTBlocks.RUBBER_LOG.asItem()).addCharcoalRecipe()
-                            .strippedLog(GTBlocks.STRIPPED_RUBBER_LOG.asItem())
-                            .wood(GTBlocks.RUBBER_WOOD.asItem())
-                            .strippedWood(GTBlocks.STRIPPED_RUBBER_WOOD.asItem())
-                            .door(GTBlocks.RUBBER_DOOR.asItem(), null)
-                            .trapdoor(GTBlocks.RUBBER_TRAPDOOR.asItem(), null)
-                            .slab(GTBlocks.RUBBER_SLAB.asItem(), null).addSlabRecipe()
-                            .fence(GTBlocks.RUBBER_FENCE.asItem(), null)
-                            .fenceGate(GTBlocks.RUBBER_FENCE_GATE.asItem(), null)
-                            .stairs(GTBlocks.RUBBER_STAIRS.asItem(), null).addStairsRecipe()
-                            .boat(GTItems.RUBBER_BOAT.asItem(), null)
-                            .chestBoat(GTItems.RUBBER_CHEST_BOAT.asItem(), null)
-                            .sign(GTBlocks.RUBBER_SIGN.asItem(), null)
-                            .hangingSign(GTBlocks.RUBBER_HANGING_SIGN.asItem(), null)
-                            .button(GTBlocks.RUBBER_BUTTON.asItem(), null)
-                            .pressurePlate(GTBlocks.RUBBER_PRESSURE_PLATE.asItem(), null)
-                            .registerAllTags()
-                            .registerAllMaterialInfo()
-                            .build(),
-                    new WoodTypeEntry.Builder(GTCEu.MOD_ID, "treated")
-                            .planks(GTBlocks.TREATED_WOOD_PLANK.asItem(), null)
-                            .door(GTBlocks.TREATED_WOOD_DOOR.asItem(), null)
-                            .trapdoor(GTBlocks.TREATED_WOOD_TRAPDOOR.asItem(), null)
-                            .slab(GTBlocks.TREATED_WOOD_SLAB.asItem(), null).addSlabRecipe()
-                            .fence(GTBlocks.TREATED_WOOD_FENCE.asItem(), null)
-                            .fenceGate(GTBlocks.TREATED_WOOD_FENCE_GATE.asItem(), null)
-                            .stairs(GTBlocks.TREATED_WOOD_STAIRS.asItem(), null).addStairsRecipe()
-                            .boat(GTItems.TREATED_WOOD_BOAT.asItem(), null)
-                            .chestBoat(GTItems.TREATED_WOOD_CHEST_BOAT.asItem(), null)
-                            .sign(GTBlocks.TREATED_WOOD_SIGN.asItem(), null)
-                            .hangingSign(GTBlocks.TREATED_WOOD_HANGING_SIGN.asItem(), null)
-                            .button(GTBlocks.TREATED_WOOD_BUTTON.asItem(), null)
-                            .pressurePlate(GTBlocks.TREATED_WOOD_PRESSURE_PLATE.asItem(), null)
-                            .material(TreatedWood)
-                            .generateLogToPlankRecipe(false)
-                            .registerMaterialInfo(false, true, true, true, true, true, true, true, true, true)
-                            .build());
+            DEFAULT_ENTRIES = new ArrayList<WoodTypeEntry>();
```

Comment:
```suggestion
            DEFAULT_ENTRIES = new ArrayList<>();
```
The generic is unnecessary during instantiation because it is implied by the declared type from earlier.